scripts/babel/preset-v9.spec.ts

```diff
@@ -0,0 +1,37 @@
+import preset from './preset-v9';
+
+describe(`babel preset v9`, () => {
+  const babelMockedApi = {
+    assertVersion: jest.fn(),
+    caller: jest.fn(),
+    env: jest.fn(),
+    cache: {},
+    version: '0',
+  } as unknown as import('@babel/core').ConfigAPI;
+  it(`should generate module-resolve alias mappings for @griffel preset`, () => {
+    const presetConfig = preset(babelMockedApi, { tsBaseConfigPath: './tsconfig.base.json' });
+
+    const griffelPreset = presetConfig.presets.find(presetItem => presetItem[0] === '@griffel')!;
+    const griffelPresetConfig = griffelPreset[1];
+
+    expect(griffelPresetConfig).toEqual({
+      babelOptions: {
+        plugins: [
+          [
+            'babel-plugin-module-resolver',
+            {
+              alias: expect.any(Object),
+            },
+          ],
+        ],
+      },
+    });
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const alias = (griffelPresetConfig as any).babelOptions.plugins[0][1].alias;
+    const aliasesEntries = Object.entries(alias);
+
+    expect(aliasesEntries[5][0]).toEqual(expect.stringContaining('^@fluentui/'));
+    expect(aliasesEntries[5][1]).toEqual(expect.stringMatching(/[../]+[a-z-/]+\/lib\/index\.js/));
```

Comment:
I can kind of follow the rest of the index accessing, but this one confuses me, what is the significance of the fifth alias entry (aliasEntry[5])?

Reply:
just randomly picking from the huge path alias object. nothing particular. it's not the most readable test I agree, but I decided to use actual "e2e" approach to stop regression within tsconfig.base.json.

I can refactor it a bit with more meaningful variable names.